### ISSUE TITLE
Patient's list

### ISSUE THREAD

**Issue by tsvetinaangelova12** [CombiRWTH/medical-staff-assessment#81]
Endpoint for the patient's list

**Comment by benCoder01** [comment]
Should be handled with `/api/stations/<int:station_id>/`. We need to update the wiki page for that, but waiting on some merges. Will update wiki with all endpoints probably tomorrow. Until then have a look at the urls.py in the backend. This is a good point to see which routes are currently updated.

Duplicate of #32
